### PR TITLE
add reports notification time

### PR DESCRIPTION
**Pull request by DFSko**

### CODE REVIEW

**Review by AlexeyShok — APPROVED** [review]

File: HappyTravel.Kanazawa/appsettings.json [hunk 1]
```diff
@@ -48,7 +48,8 @@
   },
   "Notification": {
     "Url": "/en/api/1.0/internal/bookings/notifications",
-    "ChunkSize": 5
+    "ChunkSize": 5,
+    "ReportsHour": 1
```
Comment (kirillta) [inline HappyTravel.Kanazawa/appsettings.json:52]: `ReportHour` at first, and what means `1`?

Comment (DFSko) [reply HappyTravel.Kanazawa/appsettings.json:52]: one o'clock

Comment (kirillta) [reply HappyTravel.Kanazawa/appsettings.json:52]: May we use `TimeOnly` here to improve readability?
```
"ReportsHour": "01:00"
```

Comment (DFSko) [reply HappyTravel.Kanazawa/appsettings.json:52]: We can, but ReportHour.Hour looks weird. Maybe rename to ReportTime?

Comment (kirillta) [reply HappyTravel.Kanazawa/appsettings.json:52]: Agree

**Pull request merged**